cni/installer/deployment/job-agent.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: azure-cni-installer-agent
```

Comment:
is it installer or manager? im fine with anything but can we follow same convention everywhere